cli/cmd/root.go

```diff
@@ -135,12 +137,7 @@ func NewCmdRoot() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&logFileName, "log-file-name", "cloudquery.log", "Log filename")
 
 	// Telemtry (analytics) flags
-	cmd.PersistentFlags().Bool("no-telemetry", false, "disable telemetry collection")
-	// we dont need viper support for most flags as all can be used via command line for now (we can add in the future if really necessary)
-	// the only exception is the telemetry as people might want to put in a bash starter script
-	if err := viper.BindPFlag("no-telemetry", cmd.PersistentFlags().Lookup("no-telemetry")); err != nil {
-		panic(err)
-	}
+	cmd.PersistentFlags().BoolVar(&noTelemetry, "no-telemetry", false, "disable telemetry collection (env: CQ_NO_TELEMETRY)")
 	cmd.PersistentFlags().Bool("telemetry-inspect", false, "enable telemetry inspection")
```

Comment:
Unrelated to this PR but let's drop `telemetry-inspect` and `telemtry-debug` they are just unused flags which bloat our help and make it hard to understand what are the "real" options

Reply:
Sure, done in https://github.com/cloudquery/cloudquery/pull/2904/commits/1470b1ce52dfb0d9234ff66b021e1c6508ac8480